### PR TITLE
Push test

### PR DESCRIPTION
**Pull request by dziq45**
Komentarz testowy

### CODE REVIEW

File: client/src/app/components/page/entry/entry.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (time2funk) [inline client/src/app/components/page/entry/entry.component.spec.ts:1]: ok, lets delete those **.spec** files.
we don't need them for now.

File: client/src/app/components/page/list/list.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (time2funk) [inline client/src/app/components/page/list/list.component.spec.ts:1]: delete this file

File: server/mongo.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const mongoose = require('mongoose');
```
Comment (time2funk) [inline server/mongo.js:1]: please left **TODO: refactor** here.
we will deal with this later in separate PR (Pull Request)

File: client/src/app/components/page/entry/entry.component.spec.ts (hunk 1 above)
Comment (dziq45) [reply client/src/app/components/page/entry/entry.component.spec.ts]: Spec file deleted.

File: client/src/app/components/page/list/list.component.spec.ts (hunk 2 above)
Comment (dziq45) [reply client/src/app/components/page/list/list.component.spec.ts]: Spec file deleted.

File: server/mongo.js (hunk 3 above)
Comment (dziq45) [reply server/mongo.js:2]: Done

**Pull request merged**